modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/methods_setattr_getattr_composed.mustache

```diff
@@ -20,28 +20,22 @@
                     )
             return None
 
-        path_to_item = []
-        if self._path_to_item:
-            path_to_item.extend(self._path_to_item)
-        path_to_item.append(name)
         raise ApiAttributeError(
             "{0} has no attribute '{1}'".format(
                 type(self).__name__, name),
-            path_to_item
+            [e for e in [self._path_to_item, name] if e]
         )
 
-    def __getattr__(self, name):
+    __unset_attribute_value__ = object()
```

Comment:
How about __unset_attribute_value__ = object
Then we always point to a singleton and don't need to instantiate additional empty objects

Reply:
well, somebody *could* concievably set an attribute to `object` value, `__unset_attribute_value__` will generate unique object only once per class, so i'm not worried about performance here